pagopa/io-app
# chore: [IOBP-610] Refactor `payments` feature

## Short description
This PR refactors the `payments` features to avoid confusion with the upcoming `wallet` feature

## List of changes proposed in this pull request
- Renamed routes, actions, stores and some components under `payments` feature and its sub-features

## How to test
Static checks should pass.
With the `io-dev-server` try to:
- Onboard a new payment method
- Make a payment
